content-security-policy/embedded-enforcement/allow_csp_from-header.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Embedded Enforcement: Allow-CSP-From header.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="../support/testharness-helper.sub.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "name": "Same origin iframes are always allowed.", 
+        "origin": Host.SAME_ORIGIN,
+        "csp": "style-src 'unsafe-inline'; script-src 'unsafe-inline';", 
+        "allow_csp_from": "",
```

Comment:
Can you add some more variants of this test? Invalid values, valid values that don't match the domain or port or scheme, valid values that do match, etc.